docs/data/toolpad/core/components/account/AccountWithDashboard.js

```diff
@@ -1,15 +1,63 @@
 import * as React from 'react';
+import PropTypes from 'prop-types';
 import Box from '@mui/material/Box';
-import { AppProvider, DashboardLayout } from '@toolpad/core';
+import Typography from '@mui/material/Typography';
+import { extendTheme } from '@mui/material/styles';
+import DashboardIcon from '@mui/icons-material/Dashboard';
+import { AppProvider } from '@toolpad/core/AppProvider';
+import { DashboardLayout } from '@toolpad/core/DashboardLayout';
+
+const NAVIGATION = [
```

Comment:
Yeah it's adding all this boilerplate here too but this way it matches the other examples...